src/tasks/utils/index.ts

```diff
@@ -22,3 +22,17 @@ export const transferETH: ActionType<{
 }> = async (taskArgs, hre) => {
   return utils.transferETH(hre, taskArgs.recipient, taskArgs.amount);
 };
+
+export const createGist: ActionType<{
```

Comment:
Haven't yet seen the "upload to DON" for the secrets? i.e DON Hosted secrets?
(https://docs.chain.link/chainlink-functions/tutorials/api-use-secrets)